Panic: Nil pointer error in schema/sso/dml/common.go#ParseUnionST_AdjCoordinate

## Description

When using `presentation.Open()` a nil pointer condition is encountered with certain powerpoint files.

Looking in the code it seems that it's generated from some other tooling (i'm not sure how it's generated)  but it is at:  

https://github.com/baliance/gooxml/blob/master/schema/soo/dml/common.go#L117

I have fixed this in a branch, and will submit a PR  for the fix, however i suspect if this is generated code, that the generating tool/config needs tweaking.

https://github.com/baliance/gooxml/blob/master/schema/soo/dml/common.go#L120 for example creates the empty pointer (not nil),

 But the other condition when no error is found, doesn't have that same pointer created: https://github.com/baliance/gooxml/blob/master/schema/soo/dml/common.go#L128

## Expected Behavior

Successfully returned Presentation, and/or Error

## Actual Behavior
```
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/custom-properties tgt: docProps/custom.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/customXml tgt: ppt/../customXml/item1.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/customXml tgt: ppt/../customXml/item2.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/customXml tgt: ppt/../customXml/item3.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/customXml tgt: ppt/../customXml/item4.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/notesMaster tgt: ppt/notesMasters/notesMaster1.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/tags tgt: ppt/tags/tag1.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/presProps tgt: ppt/presProps.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/viewProps tgt: ppt/viewProps.xml
2018/09/30 06:51:59 unsupported relationship type: http://schemas.openxmlformats.org/officeDocument/2006/relationships/tableStyles tgt: ppt/tableStyles.xml
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1328b34]

goroutine 26 [running]:
testing.tRunner.func1(0xc000124800)
	/usr/local/go/src/testing/testing.go:792 +0x387
panic(0x157ad00, 0x19a1030)
	/usr/local/go/src/runtime/panic.go:513 +0x1b9
baliance.com/gooxml/schema/soo/dml.ParseUnionST_AdjCoordinate(0x1991ad0, 0x1, 0xc0001a4f70, 0x1053a93, 0x158c640, 0xc00001f8a0)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/dml/common.go:128 +0x144
baliance.com/gooxml/schema/soo/dml.(*CT_GeomRect).UnmarshalXML(0xc000228a80, 0xc000122c60, 0xc0000dd200, 0x35, 0xc00021eada, 0x4, 0xc00022a000, 0x4, 0x4, 0x1f1de78, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/dml/CT_GeomRect.go:47 +0x49e
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1de78, 0xc000228a80, 0xc0001a5610, 0xc000228a80, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x15be820, 0xc000228a80, 0x199, 0xc0001a5610, 0x199, 0x15be820)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x158c640, 0xc000228a80, 0xc0001a5610, 0x0, 0x0)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/dml.(*CT_CustomGeometry2D).UnmarshalXML(0xc000212180, 0xc000122c60, 0xc0000dd200, 0x35, 0xc0001ff442, 0x8, 0x19c7030, 0x0, 0x0, 0x1f1ddb8, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/dml/CT_CustomGeometry2D.go:95 +0x507
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1ddb8, 0xc000212180, 0xc0001a5d38, 0xc000212180, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x15ca1c0, 0xc000212180, 0x199, 0xc0001a5d38, 0x199, 0x1581cc0)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x158b6a0, 0xc000212180, 0xc0001a5d38, 0x0, 0x0)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/dml.(*CT_ShapeProperties).UnmarshalXML(0xc0000e7500, 0xc000122c60, 0xc0000dd240, 0x3a, 0xc0001ff3ca, 0x4, 0xc0002120c0, 0x1, 0x1, 0xc0001a5e40, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/dml/CT_ShapeProperties.go:139 +0x17a6
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1d948, 0xc0000e7500, 0xc0001a6448, 0xc0000e7500, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x15dc0c0, 0xc0000e7500, 0x199, 0xc0001a6448, 0x199, 0xc0002120c0)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x1590100, 0xc0000e7500, 0xc0001a6448, 0x0, 0x0)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/pml.(*CT_Shape).UnmarshalXML(0xc000212030, 0xc000122c60, 0xc0000dd240, 0x3a, 0xc0001ff352, 0x2, 0x19c7030, 0x0, 0x0, 0xc0001a6550, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/pml/CT_Shape.go:95 +0xa56
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1d9e8, 0xc000212030, 0xc0001a6fb8, 0xc000212030, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x15986c0, 0xc00009e6d8, 0x196, 0xc0001a6fb8, 0xc00009e6d8, 0x196)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x1553c40, 0xc0001273b0, 0x197, 0xc0001a6fb8, 0x197, 0x15cb1c0)
	/usr/local/go/src/encoding/xml/read.go:396 +0x2686
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x154bc40, 0xc0001273b0, 0xc0001a6fb8, 0x0, 0x0)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/pml.(*CT_GroupShape).UnmarshalXML(0xc000201ef0, 0xc000122c60, 0xc0000dd240, 0x3a, 0xc0001ff236, 0x5, 0x19c7030, 0x0, 0x0, 0xc0001a70c0, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/pml/CT_GroupShape.go:78 +0x1040
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1d708, 0xc000201ef0, 0xc0001a7b28, 0xc000201ef0, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x1596e60, 0xc00009e6d0, 0x196, 0xc0001a7b28, 0xc00009e6d0, 0x196)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x1553ac0, 0xc000127338, 0x197, 0xc0001a7b28, 0x197, 0x15cb1c0)
	/usr/local/go/src/encoding/xml/read.go:396 +0x2686
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x154bb80, 0xc000127338, 0xc0001a7b28, 0x0, 0xc00009e6c8)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/pml.(*CT_GroupShape).UnmarshalXML(0xc0002018c0, 0xc000122c60, 0xc0000dd240, 0x3a, 0xc0001fede7, 0x5, 0x19c7030, 0x0, 0x0, 0xc0001a7c30, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/pml/CT_GroupShape.go:84 +0xe5b
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1d708, 0xc0002018c0, 0xc0001a8698, 0xc0002018c0, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x1596e60, 0xc00009e6a0, 0x196, 0xc0001a8698, 0xc00009e6a0, 0x196)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x1553ac0, 0xc000127218, 0x197, 0xc0001a8698, 0x197, 0x15cb1c0)
	/usr/local/go/src/encoding/xml/read.go:396 +0x2686
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x154bb80, 0xc000127218, 0xc0001a8698, 0x0, 0x0)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/pml.(*CT_GroupShape).UnmarshalXML(0xc0001bb770, 0xc000122c60, 0xc0000dd240, 0x3a, 0xc00019da2a, 0x6, 0x19c7030, 0x0, 0x0, 0x1f1d708, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/pml/CT_GroupShape.go:84 +0xe5b
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1d708, 0xc0001bb770, 0xc0001a8d88, 0xc0001bb770, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x15bf720, 0xc0001bb770, 0x199, 0xc0001a8d88, 0x199, 0x19c7030)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x1596e60, 0xc0001bb770, 0xc0001a8d88, 0x0, 0x1053b3d)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/pml.(*CT_CommonSlideData).UnmarshalXML(0xc0001bb710, 0xc000122c60, 0xc0000dd240, 0x3a, 0xc00019da22, 0x4, 0x19c7030, 0x0, 0x0, 0x1f1d6e8, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/pml/CT_CommonSlideData.go:95 +0x921
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1d6e8, 0xc0001bb710, 0xc0001a94b8, 0xc0001bb710, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x15cb0c0, 0xc0001bb710, 0x199, 0xc0001a94b8, 0x199, 0x19c7030)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x1596000, 0xc0001bb710, 0xc0001a94b8, 0x0, 0xc0001c4480)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
baliance.com/gooxml/schema/soo/pml.(*Sld).UnmarshalXML(0xc0001c4040, 0xc000122c60, 0xc0000dd240, 0x3a, 0xc00019da05, 0x3, 0xc000189500, 0x3, 0x4, 0x1f1d6c8, ...)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/schema/soo/pml/Sld.go:80 +0xa73
encoding/xml.(*Decoder).unmarshalInterface(0xc000122c60, 0x1f1d6c8, 0xc0001c4040, 0xc0001c4440, 0xc0001c4040, 0x16)
	/usr/local/go/src/encoding/xml/read.go:201 +0x99
encoding/xml.(*Decoder).unmarshal(0xc000122c60, 0x1584f40, 0xc0001c4040, 0x199, 0xc0001c4440, 0x199, 0x0)
	/usr/local/go/src/encoding/xml/read.go:343 +0x2b17
encoding/xml.(*Decoder).DecodeElement(0xc000122c60, 0x159c4a0, 0xc0001c4040, 0x0, 0xc000122c60, 0xc0001bb680)
	/usr/local/go/src/encoding/xml/read.go:146 +0x149
encoding/xml.(*Decoder).Decode(0xc000122c60, 0x159c4a0, 0xc0001c4040, 0x1f19260, 0xc0001c26e0)
	/usr/local/go/src/encoding/xml/read.go:134 +0x48
baliance.com/gooxml/zippkg.Decode(0xc0000ac370, 0x159c4a0, 0xc0001c4040, 0x0, 0x0)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/zippkg/helpers.go:42 +0x1d3
baliance.com/gooxml/zippkg.(*DecodeMap).Decode(0xc000092e40, 0xc000099c00, 0xdf, 0xe0, 0xc0000a4b40, 0x0)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/zippkg/decodemap.go:113 +0x412
baliance.com/gooxml/presentation.Read(0x1663900, 0xc00009e100, 0x55457d, 0xc00008d110, 0x0, 0x0)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/presentation/read.go:43 +0x308
baliance.com/gooxml/presentation.Open(0x15f89b4, 0x19, 0x0, 0x0, 0x0)
	/Users/scates/go/pkg/mod/baliance.com/gooxml@v1.0.0/presentation/open.go:35 +0x308
```



Please include a reproducible code snippet or document attachment that
demonstrates the issue.

--- a/schema/soo/dml/common.go
+++ b/schema/soo/dml/common.go
@@ -125,6 +125,7 @@ func ParseUnionST_AdjCoordinate(s string) (ST_AdjCoordinate, error) {
 			// geom guide name can be anything
 			r.ST_GeomGuideName = &s
 		} else {
+			r.ST_Coordinate = &ST_Coordinate{}
 			r.ST_Coordinate.ST_CoordinateUnqualified = &v
 		}
 	}

